test/framework/prometheus.go

```diff
@@ -287,7 +287,7 @@ func (f *Framework) UpdatePrometheusAndWaitUntilReady(ns string, p *monitoringv1
 	if err != nil {
 		return nil, err
 	}
-	if err := f.WaitForPrometheusReady(result, 5*time.Minute); err != nil {
+	if err := f.WaitForPrometheusReady(result, 10*time.Minute); err != nil {
```

Comment:
Is this change related to the issue, or is it something independent?

Reply:
The wait timeout was increated to pass e2e tests; which might be related to under powered nodes. 

Noticed intermittent failures, which I could not reproduce locally; thus increased the timeout for status polling.  With the polling timeout doubled, the `go test` timeout increased as well as commented above.